content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst

```diff
@@ -1,262 +1,321 @@
+indicated in the :guilabel:`Scheduled Date` field of the :abbr:`DO (Delivery Order)`. The :abbr:`MO
+(Manufacturing Order)` deadline, which is the :guilabel:`Scheduled Date` field on the **:abbr:`MO
+(Manufacturing Order)`**, is calculated as the *expected delivery date* - manufacturing lead time.
+This ensures that the manufacturing process begins on time to meet the delivery date. However it's
+important to note that lead times are based on calendar days. Lead times do not consider weekends,
+holidays, or work enter capacity.
+
+.. seealso::
+   - :ref:`Manufacturing planning <manufacturing/management/use_mps>`
+   - :ref:`Configure automatic MO scheduling with reordering rules
+     <inventory/management/reordering_rules>`
+   - call to action to enable weekends and holidays through a custom development? (is this salesy)
+
+.. example::
+   A product's scheduled shipment date on the :abbr:`DO (Delivery Order)` is August 15th. The
+   product requires 14 days to manufacture, so the latest date to start the :abbr:`MO (Manufacturing
+   Order)` to meet the commitment date is August 1st.
+
+.. _inventory/management/planning/scheduled/manuf-security-LT:
+
+Manufacturing security lead time
+--------------------------------
+
+*Manufacturing security lead time* is set globally for the business in :menuselection:`Manufacturing
+app --> Configuration --> Settings`. On the configuration page, under the :guilabel:`Planning`
+heading, locate the box for :guilabel:`Security Lead Time` to enable the feature by checking the
+box. Next, enter the desired number of calendar days. By configuring the security lead time, a
+buffer is set to account for potential delays in the manufacturing process.
+
+.. image:: scheduled_dates/manuf-security-LT.png
+   :align: center
+   :alt: View of the security lead time for manufacturing from the manufacturing app settings
+
+.. example::
+   A product has a scheduled shipment date on the :abbr:`DO (Delivery Order)` set for August 15th.
+   The manufacturing lead time is 7 days, and manufacturing security lead time is 3 days. So, the
+   :guilabel:`Scheduled Date` on the :abbr:`MO (Manufacturing Order)` reflects the latest date to
+   begin the manufacturing order. In this example, the :abbr:`MO (Manufacturing Order)`'s planned
+   date is August 5th.
+
+Global example
+==============
 
-.. image:: scheduled_dates/scheduled_dates_09.png
-    :align: center
-    :alt: View of the manufacturing lead time configuration from the product form
+Consider the following example to see how all the lead times work together to ensure timely order
+fulfillment. Configured lead times:
 
-When working with *Manufacturing Lead Times*, the *Deadline Start*
-of the *MO* is **Commitment Date - Manufacturing Lead Time**. For
-example, the MO’s deadline start date for an order having a commitment
-date on the 10th of July is June 27th.
+-  Sales security lead time: 1 day
```

Comment:
Do we wanna bold any of these bad boys in this list? So it would be like:

- **Manufacturing security lead time**: 2 days